src/xx-apk

```diff
@@ -51,6 +51,12 @@ setup() {
     echo "https://dl-cdn.alpinelinux.org/alpine/edge/main" >"$apk_dir/repositories"
     echo "https://dl-cdn.alpinelinux.org/alpine/edge/community" >>"$apk_dir/repositories"
   fi
+  # add loongarch support
+  if [ "$TARGETARCH" = "loong64" ]; then
+    echo "http://alpine.loongnix.cn/v3.11/main" >"$apk_dir/repositories"
+    echo "http://alpine.loongnix.cn/v3.11/community" >>"$apk_dir/repositories"
+    echo "http://alpine.loongnix.cn/v3.11/testing" >>"$apk_dir/repositories"
```

Comment:
v3.11 seems old

Reply:
> v3.11 seems old

v3.18 is about to be launched, is it okay to keep the status quo?

Reply:
Would it possible to have smth an edge branch like `http://alpine.loongnix.cn/edge/testing`?